test/e2e_test.go

```diff
@@ -944,7 +949,7 @@ func TestAttachSBOM(t *testing.T) {
 	// Upload it!
 	must(attach.SBOMCmd(ctx, options.RegistryOptions{}, "./testdata/bom-go-mod.spdx", "spdx", imgName), t)
```

Comment:
Should attach also take a platform? 🤔

Reply:
OK, I'll add `--platform` to `cosign sbom attach` in a follow-up